mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1328,6 +1329,25 @@ pub fn min_trading_vol(coin: &UtxoCoinFields) -> MmNumber {
 
 pub fn is_asset_chain(coin: &UtxoCoinFields) -> bool { coin.conf.asset_chain }
 
+pub async fn get_raw_transaction<T>(coin: T, req: RawTransactionRequest) -> RawTransactionResult
+where
+    T: AsRef<UtxoCoinFields> + UtxoCommonOps + MarketCoinOps,
+{
+    let hash = match H256Json::from_str(&req.tx_hash) {
+        Ok(h) => h,
+        Err(_) => {
+            return MmError::err(RawTransactionError::InvalidHashError(req.tx_hash));
```

Comment:
Please replace return statements with `?` implementing `From<...>` trait for error(s) or using `map_to_mm`, example

https://github.com/KomodoPlatform/atomicDEX-API/blob/e98c61217451f38f54ae5d60f3d3b012ab2ac962/mm2src/coins/eth.rs#L563